main/fw_update.c

```diff
@@ -36,6 +36,9 @@
 
 #define FW_UPDATE_DELAY_BEFORE_REBOOT_SECONDS (5U)
 
+#define FW_UPDATE_MAX_OTA_PARTITION_SIZE  (4U * 1024U * 1024U)
```

Comment:
Is there a way to retrieve these values off the partition table file? 

If partition table is ever changed, I'm surprised if we remember to adjust these constants

Reply:
Actually, since these constants are used only for calculating percentages in UI in case file size is not reported, I think this is ok as long as nothing breaks on values > 100%